packages/framework/tests/Unit/Component/Domain/DomainTest.php

```diff
@@ -10,28 +10,47 @@
 
 class DomainTest extends TestCase
 {
-    public function testGetIdNotSet()
+    /**
+     * @return \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig
+     */
+    private function createDomainConfigFirst(): DomainConfig
     {
-        $domainConfigs = [
-            new DomainConfig(1, 'http://example.com:8080', 'example', 'cs'),
-            new DomainConfig(2, 'http://example.org:8080', 'example.org', 'en'),
+        return new DomainConfig(Domain::FIRST_DOMAIN_ID, 'http://example.com:8080', 'example.com', 'cs');
+    }
+
+    /**
+     * @return \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig
+     */
+    private function createDomainConfigSecond(): DomainConfig
+    {
+        return new DomainConfig(Domain::SECOND_DOMAIN_ID, 'http://example.org:8080', 'example.org', 'en');
+    }
+
+    /**
+     * @return array
+     */
+    private function getDomainConfigs(): array
+    {
+        return [
+            $this->createDomainConfigFirst(),
+            $this->createDomainConfigSecond(),
         ];
+    }
+
+    public function testGetIdNotSet()
```

Comment:
```suggestion
    public function testGetIdNotSet(): void
```

Reply:
This was actually unchanged, but I will fix it anyway.

Reply:
Yes, I wrote more suggestions, but i removed it, but when I miss typehint in green backround it interested me :smile: